redhat-developer/developers.redhat.com
# [DEVELOPER-3405] Connectors assembly type

### Issue Link

Resolves #3405 

### Stakeholders

@joeschram 
@sjcox-rh 
@gdoyle1 

### Verification Process

- Go here: Any new, assembly-based product page ex. /products/openjdk/overview
- In a separate tab, open the edit page: /node/207365/edit
- In the Overview Page Content field, create a Connectors assembly.
  - This should be available from the Overview Page Content, Download Page Content and Getting Started page content fields.
- Save the node
- Refresh the Product overview page in your other tab

You should see this:

<img width="1427" alt="Screen Shot 2019-12-05 at 1 50 15 PM" src="https://user-images.githubusercontent.com/7155034/70277140-a4f76080-1766-11ea-8cb7-f56b21070cef.png">
